.gitattributes

```diff
@@ -0,0 +1,3 @@
+*.js    text eol=lf

```

Comment:
This probably isn't the best way to fix this, but I tried adding

```json
"endOfLine": "auto",
```

to _package.json_. but I was still getting lint warnings.

Reply:
Would be good to figure out how to not do this, no project I've used with prettier needs it so I'm skeptical that this is the best solution.